deployment/ccip/changeset/internal/deploy_home_chain.go

```diff
@@ -409,9 +406,24 @@ func BuildOCR3ConfigForCCIPHome(
 
 		transmittersBytes := make([][]byte, len(transmitters))
 		for i, transmitter := range transmitters {
-			parsed, err2 := common.ParseHexOrString(string(transmitter))
-			if err2 != nil {
-				return nil, err2
+			// TODO: this should just use the addresscodec
+			family, err := chain_selectors.GetSelectorFamily(destSelector)
+			if err != nil {
+				return nil, err
+			}
+			var parsed []byte
+			switch family {
+			case chain_selectors.FamilyEVM:
+				parsed, err2 = common.ParseHexOrString(string(transmitter))
+				if err2 != nil {
+					return nil, err2
+				}
+			case chain_selectors.FamilySolana:
+				pk, err := solana.PublicKeyFromBase58(string(transmitter))
+				if err != nil {
+					return nil, fmt.Errorf("failed to decode SVM address '%s': %w", transmitter, err)
+				}
+				parsed = pk.Bytes()
```

Comment:
Just confirming, but this is the actual 32 byte public key right, not the string representation?

Reply:
Yep exactly 👍🏻